src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentNavigationService.cs

```diff
@@ -275,7 +275,11 @@ private static Document OpenDocument(Workspace workspace, DocumentId documentId,
             {
                 if (options.GetOption(NavigationOptions.PreferProvisionalTab))
                 {
-                    using (NewDocumentStateScope ndss = new NewDocumentStateScope(__VSNEWDOCUMENTSTATE.NDS_Provisional, VSConstants.NewDocumentStateReason.Navigation))
+                    // If we're just opening the provisional tab, then do not "activate" the document
+                    // (i.e. don't give it focus).  This way if a user is just arrowing through a set 
+                    // of FindAllReferences results, they don't have their cursor placed into the document.
+                    var state = __VSNEWDOCUMENTSTATE.NDS_Provisional | __VSNEWDOCUMENTSTATE.NDS_NoActivate;
```

Comment:
:question: What happens in the following scenario:

1. Open a document
2. Use the Find All References feature on a local variable with two or more references
3. Close the first document and open a different document
4. Click on a reference in the Find All References window. A provisional tab should appear.
5. Click on the second document which is already open, thus deactivating but not closing the provisional tab.
6. Click the second reference. The provisional tab should be activated again but shouldn't have focus. ⬅️ I'm concerned this step might not be working.